pvlib/pvsystem.py

```diff
@@ -1223,6 +1255,146 @@ def calcparams_desoto(effective_irradiance, temp_cell,
     return IL, I0, Rs, Rsh, nNsVth
 
 
+def calcparams_cec(effective_irradiance, temp_cell,
+                   alpha_sc, a_ref, I_L_ref, I_o_ref, R_sh_ref, R_s,
+                   Adjust, EgRef=1.121, dEgdT=-0.0002677,
+                   irrad_ref=1000, temp_ref=25):
+    '''
+    Calculates five parameter values for the single diode equation at
+    effective irradiance and cell temperature using the CEC
+    model described in [1]. The five values returned by calcparams_cec
+    can be used by singlediode to calculate an IV curve.
```

Comment:
Perhaps add a comment on about how this differs from ``calcparams_desoto``. Either here or a Notes section below the Returns section would work.

Reply:
There was a comment in the Notes section below the References, but it's easy to overlook, so I moved the comment up and deleted the Notes.